### PR TITLE
feat: Add DefaultTemplatePathPrefix

### PR DESCRIPTION
**Pull request by yngvark**
# Description

This is a proof of concept for a simple way to supporting github actions templates. We can discuss other designs / variable namings before applying. Some designs are discussed in https://github.com/oslokommune/golden-path-boilerplate/issues/339.

We might want more advanced capabilities later, but right now I wanted to keep things simple.

---

Design question: Should line 1 in the file below always be required, or should it default to `boilerplate/terraform`?

https://github.com/oslokommune/ok/blob/96793563334be8a7052b12e436c17877491b4527/pkg/pkg/install/testdata/packages-github-actions.yml#L1

Given above example, `ok` will use this template URL:
The value refer to a directory under `https://github.com/oslokommune/golden-path-boilerplate/tree/main/boilerplate/github-actions`.

Currently, if this line is omitted, `boilerplate/terraform` will be used as the template path prefix. We most often work with Terraform, so this makes sense to me.

# Motivation

https://github.com/oslokommune/golden-path-boilerplate/issues/339

### CODE REVIEW

File: pkg/pkg/common/manifest.go [hunk 1]
```diff
@@ -3,7 +3,8 @@ package common
 import "fmt"
 
 type PackageManifest struct {
-	Packages []Package `yaml:"Packages"`
+	Packages                  []Package `yaml:"Packages"`
+	DefaultTemplatePathPrefix string    `yaml:"DefaultTemplatePathPrefix"`
```
Comment (haavardeide) [inline pkg/pkg/common/manifest.go:7]: wouldn't `TemplatePathPrefix` be a better name? Will `Default` add any meaning for the end user?

we can use `DefaultTemplatePathPrefix` as const in install.go, but that says something about what is default if nothing else is set

Comment (haavardeide) [reply pkg/pkg/common/manifest.go:7]: Or should this be named `PackagePath`?

Comment (yngvark) [reply pkg/pkg/common/manifest.go:7]: We agreed on `DefaultPackagePathPrefix` in meeting.

Comment (yngvark) [reply pkg/pkg/common/manifest.go:7]: A Package is template that is packaged, thus is versioned and released.

**Review by haavardeide — APPROVED** [review]
:shipit:

**Pull request merged**